practice_app/shipment_calculator/views.py

```diff
@@ -0,0 +1,50 @@
+from django.http import JsonResponse, HttpResponse
+from django.shortcuts import render, redirect
+from .google_api.utils import MapsAPI
+from .forms import ProductForm
+from .forms import VendorForm
+from .models import Product
+
+def post_vendor(request):
+    if request.method == "POST":
+        form = VendorForm(request.POST)
+        if form.is_valid():
+            try:
+                form.save()
+                return redirect('/shipment_calculator/add_vendor/')
+            except:
+                pass
+    else:
+        form = VendorForm()
+    return render(request, 'shipment_calculator/add_vendor.html', {'form': form})
```

Comment:
I couldn't find the add_vendor.html. Did I miss it?

Reply:
I see it here 'practice_app/shipment_calculator/templates/shipment_calculator/add_vendor.html'.